### PR TITLE
Fixes cows insta-reacting with their own milk

### LINKED ISSUE #11035: Cow milking onto the floor doesn't work
### Labels

[BUG][MINOR]

### Description

If you *milk onto the floor you lose the blood but no puddle appears. You also get the chat message implying it worked.

### Steps to reproduce

1. Be cow
2. say *milk
3. Nothing happened

### Expected Behavior

Milk puddle

### Additional Information & Screenshots

Reported by: Asche
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-09-26-22-58
Reported on: 2022-09-26 23:12:38

### PR DESCRIPTION
**Pull request by MetricDuck**
[BUG]
## About the PR 
When cows *milk themselves they should leave a puddle of milk on the floor at their feet as it falls directly out of their bloodstream. However, current behaviour is bugged due to the milk instantly and completely reacting with the cow as a touch reaction with a volume fraction of 1 thereby seemingly preventing the milk from ever actually reaching the floor. 

This occurs because reaction_mob() is calling it's parent instead of returning it like all other touch chems.

This PR fixes the bugged behaviour by making reaction_mob() return its parent instead.

## Why's this needed? 
Fixes #11035

### CODE REVIEW

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@ datum
 				return
 
 			reaction_mob(var/mob/M, var/method=TOUCH, var/volume, var/paramslist = 0)
-				..()
+				. = ..()
 				if(M.mob_flags & IS_BONEY)
 					M.HealDamage("All", clamp(1 * volume, 0, 20), clamp(1 * volume, 0, 20)) //put a cap on instant healing
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-FoodDrink.dm:74]: should set `. = FALSE` here though, since it's being used for a touch reaction at this point

Comment (frawhst) [reply code/modules/chemistry/Reagents-FoodDrink.dm:74]: you can splash milk on skeletons to heal them??? wild

Comment (Tarmunora) [reply code/modules/chemistry/Reagents-FoodDrink.dm:74]: I did not know this until now either

Comment (Tarmunora) [reply code/modules/chemistry/Reagents-FoodDrink.dm:74]: looks like it was changed in #9977. hm

Comment (MetricDuck) [reply code/modules/chemistry/Reagents-FoodDrink.dm:75]: Amended

Would note the following:
- Healing triggers on touch or ingest, unless the reagent cap is full in which case touch only. The cap is reached very rapidly with critters so they need to transfer milk to a beaker to use it properly. Skeleton race gets the benefit of both.
- Double dipping with bleed reagent transfer to heal yourself is a thing but seemed to be net negative when testing.

**Pull request merged**